src/Yoti.Auth/YotiClientEngine.cs

```diff
@@ -118,23 +98,23 @@ public async Task<AmlResult> PerformAmlCheckAsync(string appId, AsymmetricCipher
 
             string endpoint = EndpointFactory.CreateAmlEndpoint(httpMethod, appId);
 
-            Dictionary<string, string> headers = CreateHeaders(keyPair, httpMethod, endpoint, httpContent, contentType: YotiConstants.ContentTypeJson);
+            Dictionary<string, string> headers = CreateHeaders(keyPair, httpMethod, endpoint, httpContent);
 
             AmlResult result = await Task.Run(async () => await new RemoteAmlService()
                 .PerformCheck(_httpRequester, amlProfile, headers, apiUrl, endpoint, httpContent));
 
             return result;
         }
 
-        private static Dictionary<string, string> CreateHeaders(AsymmetricCipherKeyPair keyPair, HttpMethod httpMethod, string endpoint, byte[] httpContent, string contentType)
+        public static Dictionary<string, string> CreateHeaders(AsymmetricCipherKeyPair keyPair, HttpMethod httpMethod, string endpoint, byte[] httpContent)
```

Comment:
Not sure we want to expose this as _public_...?

Reply:
Hmm yes not sure why that happened, will change